tests/assert_pass/keccak256_generic.yul

```diff
@@ -0,0 +1,9 @@
+{
+    mstore(0, calldataload(0))
+    let x := keccak256(0, calldataload(7))
+//    mstore(0, calldataload(9))
```

Comment:
Without this the test is failing, although the condition should actually ensure that the memory contents are the same, isn't it?